app/Http/Controllers/PostsController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\Post\PostRequest;
+use App\Models\Category;
+use App\Models\Post;
+use Throwable;
+
+class PostsController extends Controller
+{
+    public function index()
+    {
+        $posts = Post::query()->get();
+
+        return view("sections.posts.posts-page", ["posts" => $posts]);
+    }
+
+    public function show(int $id)
```

Comment:
ta usando o nome errado aqui, seria o `edit`?
```suggestion
    public function edit(int $id)
```